TASK/SetDriveTaskDrivingStyle.md

```diff
@@ -8,16 +8,81 @@ ns: TASK
+
+```c
+enum VehicleDrivingFlags
+{
+  None = 0,
+  StopForVehicles = 1,
+  StopForPeds = 2,
+  SwerveAroundAllVehicles = 4,
+  SteerAroundStationaryVehicles = 8,
+  SteerAroundPeds = 16,
+  SteerAroundObjects = 32,
+  DontSteerAroundPlayerPed = 64,
+  StopAtTrafficLights = 128,
+  GoOffRoadWhenAvoiding = 256,
+  AllowGoingWrongWay = 512,
+  Reverse = 1024,
+  UseWanderFallbackInsteadOfStraightLine = 2048,
+  AvoidRestrictedAreas = 4096,
+  PreventBackgroundPathfinding = 8192,
+  AdjustCruiseSpeedBasedOnRoadSpeed = 16384,
+  UseShortCutLinks = 262144,
+  ChangeLanesAroundObstructions = 524288,
+  UseSwitchedOffNodes = 2097152,
+  PreferNavmeshRoute = 4194304,
+  PlaneTaxiMode = 8388608,
+  ForceStraightLine = 16777216,
+  UseStringPullingAtJunctions = 33554432,
+  TryToAvoidHighways = 536870912,
+  ForceJoinInRoadDirection = 1073741824,
+  StopAtDestination = 2147483648,
+  // StopForVehicles | StopForPeds | SteerAroundObjects | SteerAroundStationaryVehicles | StopAtTrafficLights | UseShortCutLinks | ChangeLanesAroundObstructions
+  DrivingModeStopForVehicles = 786603,
+  // StopForVehicles | StopForPeds | StopAtTrafficLights | UseShortCutLinks
+  DrivingModeStopForVehiclesStrict = 262275,
+  // SwerveAroundAllVehicles | SteerAroundObjects | UseShortCutLinks | ChangeLanesAroundObstructions | StopForVehicles
+  DrivingModeAvoidVehicles = 786469,
+  // SwerveAroundAllVehicles | SteerAroundObjects | UseShortCutLinks | ChangeLanesAroundObstructions
+  DrivingModeAvoidVehiclesReckless = 786468,
+  // StopForVehicles | SteerAroundStationaryVehicles | StopForPeds | SteerAroundObjects | UseShortCutLinks | ChangeLanesAroundObstructions
+  DrivingModeStopForVehiclesIgnoreLights = 786475,
+  // SwerveAroundAllVehicles | StopAtTrafficLights | SteerAroundObjects | UseShortCutLinks | ChangeLanesAroundObstructions | StopForVehicles
+  DrivingModeAvoidVehiclesObeyLights = 786597,
+  // SwerveAroundAllVehicles | StopAtTrafficLights | StopForPeds | SteerAroundObjects | UseShortCutLinks | ChangeLanesAroundObstructions | StopForVehicles
+  DrivingModeAvoidVehiclesStopForPedsObeyLights = 786599,
+}
 ```
 
 ## Parameters
-* **ped**: 
-* **drivingStyle**: 
+* **ped**: The ped to have their driving style set.
+* **drivingStyle**: The driving style (see `VehicleDrivingFlags`).
+
+## Examples
 
+```lua
+local vehicle_model = `adder`
+RequestModel(vehicle_model)
+repeat Wait(0) until HasModelLoaded(vehicle_model)
+
+local ped = PlayerPedId() -- Player needs to be in a vehicle for this to work
+local coords = GetEntityCoords(ped) - GetEntityForwardVector(ped) * 15.0
```

Comment:
Is this the order of operation you intended?

Reply:
Yes as the vehicle is chasing the player ped, so I'm spawning that vehicle behind the player ped.